<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookCitationWindow</class>
 <widget class="QMainWindow" name="BookCitationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>281</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Input data for citation</string>
    </property>
    <widget class="QLabel" name="label_place">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Place</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineedit_publisher">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>146</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_title">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>20</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineedit_title">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>134</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_year">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>22</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Year</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineedit_year">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>136</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_author">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>33</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Author</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineedit_author">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>147</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineedit_place">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_publisher">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Publisher</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_result">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>291</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>320</y>
      <width>168</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="button_copy">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Copy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_go">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Go!</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="button_back">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:#000000;
}

QPushButton:Pressed{
background-color:#24c8c8;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>337</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QRadioButton" name="radio_1st">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>One Author</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radio_2nd">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>2nd Author</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radio_3rd">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>3rd Author</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_copy">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>300</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_reset">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>41</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>R
e
s
e
t</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>357</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineedit_author</tabstop>
  <tabstop>lineedit_year</tabstop>
  <tabstop>lineedit_title</tabstop>
  <tabstop>lineedit_place</tabstop>
  <tabstop>lineedit_publisher</tabstop>
  <tabstop>button_go</tabstop>
  <tabstop>radio_1st</tabstop>
  <tabstop>radio_2nd</tabstop>
  <tabstop>radio_3rd</tabstop>
  <tabstop>button_copy</tabstop>
  <tabstop>button_back</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
